core/src/client/mod.rs

```diff
@@ -522,16 +522,6 @@ impl<T: Clone> RequestIdGuard<T> {
 	}
 }
 
-/// What certificate store to use
```

Comment:
remove this from here because I don't want to expose another feature flag for TLS stuff